pkg/server/plugin/upstreamauthority/certmanager/certmanager_test.go

```diff
@@ -51,112 +54,88 @@ func Test_MintX509CA(t *testing.T) {
 	intermediate, intermediatePEM := testingCAPEM(t)
 
 	tests := map[string]struct {
-		request              *upstreamauthorityv0.MintX509CARequest
+		csr                  []byte
+		preferredTTL         time.Duration
 		updateCR             func(t *testing.T, cr *cmapi.CertificateRequest)
-		expX509CaChain       [][]byte
-		expUpstreamX509Roots [][]byte
-		expError             bool
+		expX509Ca            []*x509.Certificate
+		expUpstreamX509Roots []*x509.Certificate
```

Comment:
can we name these consistent with `expectCode` and `expectMsgPrefix`? i.e. expectX509CA and expectUpstreamX509Roots